app/src/main/kotlin/com/mapzen/erasermap/view/MainActivity.kt

```diff
@@ -164,8 +170,20 @@ public class MainActivity : AppCompatActivity(), MainViewController, RouteCallba
     private fun initFindMeButton() {
         findMe = MapData("find_me")
         Tangram.addDataSource(findMe);
-        findMeButton?.visibility = View.VISIBLE
-        findMeButton?.setOnClickListener({ presenter?.onFindMeButtonClick() })
+        findMeButton.visibility = View.VISIBLE
+        findMeButton.setOnClickListener({ presenter?.onFindMeButtonClick() })
```

Comment:
Property access syntax when calling from Kotlin into Java was added after we started the app so a lot of those changes are just updating old code.

In this case Kotlin is unable to use property access syntax because `View.setOnClickListener(...)` is not a true setter - it does not actually set a field on itself but rather invokes 'getListenerInfo().mOnClickListener`.

https://android.googlesource.com/platform/frameworks/base/+/refs/heads/master/core/java/android/view/View.java#5126